ReleaseNotesGenerator/NewRelease.cs

```diff
@@ -1,13 +1,13 @@
-using Octokit;
-using System;
-using Microsoft.WindowsAzure.Storage;
+using Azure.Storage.Blobs;
+using Microsoft.AspNetCore.Http;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.Http;
-using Microsoft.AspNetCore.Http;
 using Microsoft.Azure.WebJobs.Host;
-using System.Threading.Tasks;
-using System.IO;
 using Newtonsoft.Json;
+using Octokit;
```

Comment:
What's this include used for?

Reply:
![Screenshot 2021-12-23 095320](https://user-images.githubusercontent.com/87058034/147175962-566355ca-8d8d-4c3b-9765-78e0b550a81a.png)
 These underlined in red all need this include.

Reply:
Oh interesting that it hasn't thrown an error before this PR. Since all those underlined in red aren't apart of this PR.